scratch/phase-00-lesson-05/exercise-1.ipynb

```
@@ -0,0 +1,162 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "73410a8a-edf6-424c-b671-9e496c0da632",
+   "metadata": {},
+   "outputs": [
+    {
```

Ruff (0.15.12)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)